backend/src/route/schemas.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class Message(BaseModel):
+    detail: str
+
+
+class Response500(BaseModel):
+    model_config = {
+        "json_schema_extra": {
+            "title": "Server error",
+            "description": "Server error",
+            "example": {
+                "detail": "Server error",
+            },
+        },
+    }
+
+
+class Response404(BaseModel):
+    model_config = {
+        "json_schema_extra": {
+            "title": "Not found",
+            "description": "Not found",
+            "example": {
+                "detail": "Not found",
+            },
+        },
+    }
```

Comment:
Im not sure about it. What is the purpose ?

Reply:
I'd remove it

Reply:
its for schema in /docs